x/stakeibc/keeper/msg_server_submit_tx.go

```diff
@@ -171,30 +171,30 @@ func (k Keeper) UpdateWithdrawalBalance(ctx sdk.Context, zoneInfo types.HostZone
 }
 
 // SubmitTxs submits an ICA transaction containing multiple messages
-func (k Keeper) SubmitTxs(ctx sdk.Context, connectionId string, msgs []sdk.Msg, account types.ICAAccount) error {
+func (k Keeper) SubmitTxs(ctx sdk.Context, connectionId string, msgs []sdk.Msg, account types.ICAAccount) (uint64, error) {
 	chainId, err := k.GetChainID(ctx, connectionId)
 	if err != nil {
-		return err
+		return 0, err
```

Comment:
I assume we're error checking everytime we call SubmitTxs so this return val of 0 shouldn't be an issue. Is there a safer val we can return here? Don't think so (negatives and nils won't work).

Reply:
Either should be fine!